plugins/modules/remote_management/redfish/redfish_command.py

```diff
@@ -63,28 +63,28 @@
     required: false
     aliases: [ account_username ]
     description:
-      - Username of account to add/delete/modify
+      - Username of account to add/delete/modify.
     type: str
   new_password:
     required: false
     aliases: [ account_password ]
     description:
-      - New password of account to add/modify
+      - New password of account to add/modify.
     type: str
   roleid:
     required: false
     aliases: [ account_roleid ]
     description:
-      - Role of account to add/modify
+      - Role of account to add/modify.
     type: str
   bootdevice:
     required: false
     description:
-      - bootdevice when setting boot configuration
+      - Bootdevice when setting boot configuration.
```

Comment:
Is bootdevice (without space) a domain-specific term, or should it be two words?
```suggestion
      - Boot device when setting boot configuration.
```